packages/libmatio/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/tbeu/matio"
+TERMUX_PKG_DESCRIPTION="A C library for reading and writing Matlab MAT files"
+TERMUX_PKG_GROUPS="science"
+TERMUX_PKG_LICENSE="BSD 2-Clause"
+TERMUX_PKG_LICENSE_FILE="COPYING"
```

Comment:
This line shouldn't be needed, COPYING is one of the filenames [termux_step_install_license looks for](https://github.com/termux/termux-packages/blob/master/scripts/build/termux_step_install_license.sh#L35-L42)